blocked-edges/4.14.0-ARODNSWrongBootSequence.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.14.0
+from: .*
+url: https://issues.redhat.com/browse/OCPBUGS-35300
+name: ARODNSWrongBootSequence
+message: |-
+  Disconnected clusters or clusters with a UDR 0.0.0.0/0 route definition that are blocking the ARO ACR and quay, are not be able to add or replace nodes after an upgrade
```

Comment:
nit: can we open with "Disconnected ARO clusters or ARO clusters with..." just to be extra-clear that this is an ARO declaration?

Reply:
2c351d82177ebfe1 uses `Disconnected ARO clusters or clusters with`, which is close enough for me; marking this thread resolved.